src/Dots.js

```diff
@@ -74,7 +83,7 @@ export default class Dots extends Component {
                 circle
                 zDepth={0}
                 style={{
-                  ...styles.dot,
+                  ...dotStyle,
```

Comment:
Just using
```
...styles.dot,
color: dotColor
```
would make all the weird state stuff and the added prepare function unneccessary.